CMakeLists.txt

```diff
@@ -64,6 +67,10 @@ endif()
 # PIC (Position Independent Code) ensures .a files can be linked to executables that have PIE enabled
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+if (DO_ENABLE_EXCEPTIONS)
+    #TODO(jimson): There is no cmake option for disabling exceptions, so we must set a compiler-specific flag if we want to enforce disabling exceptions
```

Comment:
>disabling [](http://example.com/codeflow?start=132&length=9)

Does the chronium build really disable exceptions in the compiler? IMO we should not worry about it since the builder can configure cmake over command line and set whatever compiler options they want.